assets/src/edit-story/components/canvas/elementEdit.js

```diff
@@ -0,0 +1,57 @@
+/**
+ * External dependencies
+ */
+import PropTypes from 'prop-types';
+import styled from 'styled-components';
+
+/**
+ * Internal dependencies
+ */
+import { getDefinitionForType } from '../../elements';
+import { ElementWithPosition, ElementWithSize, ElementWithRotation, getBox } from '../../elements/shared';
+
+// Background color is used to make the edited element more prominent and
+// easier to see.
+const Wrapper = styled.div`
+  ${ ElementWithPosition }
+  ${ ElementWithSize }
+  ${ ElementWithRotation }
+  pointer-events: initial;
+	background-color: rgba(255, 255, 255, 0.5);
```

Comment:
move to theme

Reply:
Ditto, theme vs shading.

Reply:
Ditto from above – it's used for shading, but effectively you are shading by applying white at 50% opacity on top of something else. If we don't use these in other modules though, then I'm fine with defining them as constants locally – that way we can still easily change the values if need to be.

Reply:
Ok. Agree.

Reply:
To clarify though: one is grayout and the other one is whiteout. There's really not much in-between. The only thing that could be a bit different here is the transparency. I'll start with these two as named theme colors and we'll see from there.

Reply:
yeah, you're right. I guess the color isn't something we would want to change here, realistically. The opacity value more so.